wafer/pages/models.py

```diff
@@ -68,6 +78,24 @@ def get_absolute_url(self):
         url = "/".join(self.get_path())
         return reverse('wafer_page', args=(url,))
 
+    def cached_render(self):
+        if self.cache_time < 0:
+            return self.content.rendered
+        cache = caches[self.cache_name]
+        cache_key = self.get_absolute_url()
```

Comment:
Maybe something like ``wafer.pages:rendered:...``? Just thinking that the module name is a nice existing namespace and `:` isn't a valid part of a module name.